source/installation/file-location.txt

```diff
@@ -0,0 +1,74 @@
+.. _file-location:
+
+==================================
+Relational Migrator File Locations
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Review the location of files needed when working with Relational Migrator. 
+Their location varies according to the OS you are working with. 
+
+Mac
+---
+
+The file locations for the Mac OS:
+
+- **Configuration file**
+  
+  ``~/Library/Application Support/MongoDB/Relational Migrator/user.properties``
+
+- **Driver downloads**
+
+  ``~/Library/Application Support/MongoDB/Relational Migrator/Drivers``
+
+- **Log information**
+  
+  ``~/Library/Application Support/MongoDB/Relational Migrator/Logs/migrator.log``
+
+Windows
+-------
+
+The file locations for the Windows OS:
+
+- **Configuration file**
+
+  ``c:\Users\<username>\AppData\Local\MongoDB\Relational Migrator\Data\user.properties``
```

Comment:
This path is correct, but in staging this is path is too long to fully display without horizontal scrolling. Is there a better way of showing this (wrapping on one of the spaces would be preferable to horizontal scrolling).

Reply:
I tried breaking up the line on the spacing but it unfortunately renders the same.